core/src/utils/trapezoid_profile.cpp

```diff
@@ -0,0 +1,121 @@
+#include "../core/include/utils/trapezoid_profile.h"
```

Comment:
```suggestion
#include "../core/include/utils/trapezoid_profile.h"
#include "../core/include/utils/math_util.h"
```
Add the math util file to to access sign().